src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAuthenticationController.java

```diff
@@ -116,10 +121,14 @@ public JsonResult login(@RequestBody CommonAuthenticationRequest authenticationR
             throws AuthenticationException {
 
         JsonResult jsonResult;
+        String username = authenticationRequest.getUsername();
         try {
+            if (loginAttemptService.isBlocked(username)){
+                throw new PermissionDeniedException("Try to login later");
```

Comment:
"You have exceeded the number of allowed login attempts. Please try again later."